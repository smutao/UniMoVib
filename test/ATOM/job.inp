Atomic thermochemistry calculation (Ne atom)
The total energy was calculated at the HF/STO-3G level

 $contrl
   qcprog="atomcalc"
 $end

 $Thermo
   Eel=-126.604525089
 $end

 $IsoMas $End
 19.99244

