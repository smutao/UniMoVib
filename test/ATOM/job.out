 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 Atomic thermochemistry calculation


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature          :   298.150000 Kelvin
 Pressure             :     1.000000 Atm
 Atomic mass          :    19.992440 AMU
 Elec. total energy   :  -126.604525 Hartree

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.000000            0.000000
 Thermal correction to Energy               :            0.001416            0.888728
 Thermal correction to Enthalpy             :            0.002360            1.481214
 Thermal correction to Gibbs Free Energy    :           -0.014231           -8.930035

 Sum of electronic and zero-point Energies  :         -126.604525
 Sum of electronic and thermal Energies     :         -126.603109
 Sum of electronic and thermal Enthalpies   :         -126.602165
 Sum of electronic and thermal Free Energies:         -126.618756


 Job terminated correctly! Tue May 02 15:57:51 2017
